bvm/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/CPU.java

```diff
@@ -441,7 +442,8 @@ private static void tryExec(WorkerExecutionContext ctx) {
                         i = operands[0];
                         j = operands[1];
                         k = operands[2];
-                        retryTransaction(ctx, i, j, k);
+                        l = operands[3];
+//                        retryTransaction(ctx, i, j, k, l);
```

Comment:
Why this is commented here?

Reply:
Since CPU class is no longer used, they were commented out to fix compilation failures.
Do you think it's ok to remove the function them self, for example remove retryTransaction function.

Reply:
Agreed. Lets keep them as it is then. It will get fixed with the removal of CPU class.